README.md

```diff
@@ -1,57 +1,149 @@
 # React createElement()
 
 ## Objectives
-
 1. Describe how to call `React.createElement()`
 2. Describe how we can build elements out of other React elements
 
-## Overview
+## Baby steps
+![Baby steps](https://media.giphy.com/media/2D4tYGhHKFYre/giphy.gif)
 
-First things first, have students include React in this lab: add
+Let's write some code! Before we start though, we need to set some things up. We're going to use the React (and
+ReactDOM) libraries, so we should include those in our code. Let's do that now by referencing the right files in our
+`index.html` file:
 
 ```html
 <script src="node_modules/react/dist/react.js"></script>
 <script src="node_modules/react-dom/react-dom.js"></script>
 ```
```

Comment:
I can only get the first step here to run if I put these lines into the `<head>`...